src/main/resources/interface-specification.yml

```diff
@@ -143,6 +149,56 @@ paths:
             schema:
               $ref: '#/components/schemas/PurposeSeed'
         required: true
+  /purposes/eservice/{eServiceId}:
```

Comment:
@beetlecrunch do you think it could clash with `/purposes/{purposeId}/versions`?

Reply:
Honestly, I don't know, but this means that we have already some clashes:
- POST -> /purposes/{purposeId}/versions
- POST -> /purposes/{purposeId}/clone

Reply:
`/purposes/{purposeId}/versions`, `/purposes/{purposeId}/versions` and `/purposes/{purposeId}/clone` should not clash between them, because given a value for `purposeId` the endpoint is unique.

My concern is that in `/purposes/eservice/{eServiceId}`, `eservice` may be interpreted as `{purposeId}` of other endpoints, and it may then return a 404.
I don't remember if route check is greedy or not (but I think it is)

Reply:
Uhmmm ok.
Does {eserviceId} contribute to the uniqueness?
Anyway, let's choose another path (on BFF too).

Maybe applying this pattern:
- `POST /receive/purposes`, moving the eserviceId in the payload
- `POST /deliver/purposes`

What do you think?

Reply:
I'm gonna be a pain in the neck now, but `receive` in the context of purposes may actually correspond to `deliver` in the context of eservices (vice versa for the other) 🥲 

But I do not have suggestions if not `/reverse/purposes` or `/reversePurposes`, which is not that great either.

Feel free to pick the solution you prefer

Reply:
I got your point.
Ok for `POST -> /reverse/purposes`, moving the `eserviceId` in the payload